packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -232,3 +235,10 @@ def log_context(logger: logging.Logger, level: int, msg: str, *args, **kwargs):
     logger.log(level, "Starting " + msg + " ...", *args, **kwargs)
     yield
     logger.log(level, "Finished " + msg, *args, **kwargs)
+
+
+def get_extra(input_dict: dict) -> dict | None:
```

Comment:
I see now how this is used. I would go for something like the following
```suggestion
def get_extra(**kwargs: Any) -> dict | None:
```

See below suggested usage

Reply:
Another alternative would be
```suggestion
def get_extra(user_id: UserID | None = None) -> dict | None:
```
This way it is typed, it can always ne extended in the future and you have very little overhead when calling this.
I would have all future additional parameters observe this rule, so people know that all of them are optional.

Reply:
For now, I will put `user_id: int | str | None` as the format is not standardized everywhere if it is ok.